runtime/neumann/src/lib.rs

```diff
@@ -727,7 +727,7 @@ impl pallet_democracy::Config for Runtime {
 }
 
 parameter_types! {
-	pub const MaxTasksPerSlot: u32 = 256;
+	pub const MaxTasksPerSlot: u32 = 576;
```

Comment:
don't have to bump spec_version in this PR, right? we'll just do in the next release